packaging_uom/readme/HISTORY.rst

```diff
@@ -0,0 +1,19 @@
+0.1 (2015-08-04)
+~~~~~~~~~~~~~~~~
+
+* rename packaging_extended to packaging_uom
+
+10.0.1.0.0 (2017-03-09)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* [MIG] packaging_uom, purchase_packaging, sale_packaging: Migrated to 10.0
+
+11.0.1.0.0 (2018-08-28)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* [MIG] packaging_uom: Migration to 11.0
+
+12.0.1.0.0 (2019-05-27)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* [MIG] packaging_uom: Migration to 12.0
```

Comment:
I don't know if we should keep this file but you could add the new migration here